style/main.css

```diff
@@ -24,7 +24,7 @@ body{
     background-color: #0be941;
     box-shadow: 0 0 4px rgba(0,0,0,0.2);
     display: flex;
-    min-height:93vh;
+    margin-top: 0;

```

Comment:
Instead of setting a min height, just use non-auto margin-top so that this flex-item (width-wrap is a flex item inside flex container `<body>`) doesn't distribute extra space to the top margin